pkg/bootstrap/bootstrap_config.go

```diff
@@ -52,19 +52,52 @@ const (
 	lightstepAccessTokenBase = "lightstep_access_token.txt"
 
 	// statsPatterns gives the developer control over Envoy stats collection
-	EnvoyStatsMatcherInclusionPatterns = "sidecar.istio.io/statsInclusionPrefixes"
-)
+	EnvoyStatsMatcherInclusionPrefixes = "sidecar.istio.io/statsInclusionPrefixes"
+
+	// Options are used in templates
+	envoyStatsMatcherInclusionPrefixOption = "inclusionPrefix"
+	envoyStatsMatcherInclusionSuffixOption = "inclusionSuffix"
+
+	// statsPatterns gives the developer control over Envoy stats collection
+	EnvoyStatsMatcherInclusionSuffixes = "sidecar.istio.io/statsInclusionSuffixes"
```

Comment:
Can you register this annotation as is done in https://github.com/istio/istio/blob/master/pkg/bootstrap/bootstrap_config.go#L59 ?